recipes/libgit2/meta.yaml

```diff
@@ -0,0 +1,163 @@
+{% set name = "libgit2" %}
+{% set version_major   = "0" %}
+{% set version_minor   = "24" %}
+{% set version_revison = "0" %}
+{% set version = ".".join([version_major, version_minor, version_revison]) %}
+{% set version_tag = "0.24.1" %}
```

Comment:
The tag was not updated in the code, https://github.com/libgit2/libgit2/issues/3946. So, git tag and the resulting library name don't match.